### PR TITLE
Newlocationpage

### LINKED ISSUE #1294: New location page (anonymous flow)
## Summary

Two different location pages:
https://www.figma.com/file/R054FzUOrpP13oTbMsQevj/Sprint-5?node-id=1984%3A0
 
-- One that asks for city if reporting anonymously
(followed by review page)

Screen Shot 2020-02-10 at 3 58 03 PM
Screen Shot 2020-02-10 at 3 58 11 PM

-- One asking for postal code if not anonymous
(followed by contact details page)

Screen Shot 2020-02-06 at 9 29 46 AM

Screen Shot 2020-02-06 at 9 29 56 AM

## Unresolved questions

Will need to adjust the flow accordingly

### PR DESCRIPTION
**Pull request by DianeLiu2019**
Fixes #1294(issue)

# Description
initiate location page for report anonymously
> Is there anything related to this that still needs to be done (ex: documentation changes).
need make the page have correct flow
# Checklist:

- [ x] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [ x] I have added and needed tests for my changes (in particular for new screens)

### CODE REVIEW

File: f2/src/forms/LocationAnonymousInfoForm.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    }
+  }
+
+  return (
+    <Form
+      initialValues={location}
+      onSubmit={props.onSubmit}
+      render={({ handleSubmit }) => (
+        <Stack as="form" onSubmit={handleSubmit} shouldWrapChildren spacing={6}>
+          <Field name="City">
```
Comment (amazingphilippe) [inline f2/src/forms/LocationAnonymousInfoForm.js]: Can we use the Field component from our own components? You can checkout our Storybook docs for usage!

File: f2/src/forms/LocationAnonymousInfoForm.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                <TextInput
+                  id="City"
+                  name={props.input.name}
+                  value={props.input.value}
+                  onChange={props.input.onChange}
+                />
+              </FormControl>
+            )}
+          </Field>
+          <Field name="province">
```
Comment (amazingphilippe) [inline f2/src/forms/LocationAnonymousInfoForm.js]: Same thing here for Field

File: f2/src/LocationPage.js [hunk 3]
```diff
@@ -31,7 +31,9 @@ export const LocationPage = () => {
               <LocationInfoForm
                 onSubmit={data => {
                   dispatch({ type: 'saveFormData', data: { location: data } })
-                  history.push(doneForms ? '/confirmation' : '/contactinfo')
+                  history.push(
+                    doneForms ? '/confirmation' : '/locationAnonymous',
```
Comment (amazingphilippe) [inline f2/src/LocationPage.js]: I don't think we want to show both location+locationAnonymous pages. We only want to see one of them, depending on what the user chose on the anonymousInfo page.

Comment (DianeLiu2019) [reply f2/src/LocationPage.js]: Yes, that is right. That's why i created issue #1686 to correct the flow. if I don't put them in the current flow. People can't see both of the pages.

Comment (amazingphilippe) [reply f2/src/LocationPage.js]: Ok I understand. I suggest we leave it out of the flow. People can access it by doing `npm run dev` and accessing the page directly. Then we'll fix the flow in #1686!!

File: f2/src/locales/en.json [hunk 4]
```diff
@@ -356,7 +362,9 @@
   "whenDidItStart.startDate.warning": "Start date must contain a real day",
   "whenDidItStart.startDay": "Day",
   "whenDidItStart.startMonth": "Month",
+  "whenDidItStart.startYear": "Year"
```
Comment (amazingphilippe) [inline f2/src/locales/en.json]: missing comma

File: f2/src/locales/fr.json [hunk 5]
```diff
@@ -356,7 +362,9 @@
   "whenDidItStart.startDate.warning": "La date de l'incident doit comprendre un vrai jour",
   "whenDidItStart.startDay": "Jour",
   "whenDidItStart.startMonth": "Mois",
+  "whenDidItStart.startYear": "Année"
```
Comment (amazingphilippe) [inline f2/src/locales/fr.json]: This slipped in from a merge. Remove the line. It's defined two lines below

File: f2/src/locales/en.json (hunk 4 above)
Comment (amazingphilippe) [inline f2/src/locales/en.json]: This slipped in from a merge. Remove the line. It's defined two lines below

File: f2/src/forms/LocationAnonymousInfoForm.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
+/** @jsx jsx */
+import PropTypes from 'prop-types'
+import { jsx } from '@emotion/core'
+import { Trans } from '@lingui/macro'
+import { Form } from 'react-final-form'
+import { NextAndCancelButtons } from '../components/next-and-cancel-buttons'
+import { TextInput } from '../components/TextInput'
+import { Stack, FormControl } from '@chakra-ui/core'
+import { FormLabel } from '../components/FormLabel'
```
Comment (amazingphilippe) [inline f2/src/forms/LocationAnonymousInfoForm.js]: Make sure to lint our unused vars!

**Pull request merged**